app/styles/modules/_settings.scss

```diff
@@ -578,7 +577,7 @@ section.modal-panel {
   .last-connected {
     color: $color-grey;
     font-size: 12px;
-    margin-right: 120px;
+    margin-right: 110px;
```

Comment:
The previous margin here was trying to align with the ellipsis, now we don't need to worry about that both elements can borrow an extra 10 pixels.